### PR TITLE
Check cpu status and drivers in win checkpoints

### PR DESCRIPTION
**Pull request by mxie91**

### CODE REVIEW

**Comment by mxie91** [comment]
@xiaodwan pls help to review, thanks!

File: provider/v2v_vmcheck_helper.py [hunk 1]
```diff
@@ -786,8 +786,11 @@ def check_windows_vm(self):
         LOG.info("Checking VirtIO drivers and display adapter")
         expect_video = self.get_expect_video_model()
         has_virtio_win, has_qxldod = self.get_virtio_win_config()
+        cpu_drivers = ['Intel Processor', 'AMD Processor']
         expect_drivers = ["Red Hat VirtIO SCSI",
-                          "Red Hat VirtIO Ethernet Adapte"]
+                          "Red Hat VirtIO Ethernet Adapte",
+                          "VirtIO RNG",
+                          "VirtIO Serial"]
```
Comment (xiaodwan) [inline provider/v2v_vmcheck_helper.py:793]: I'm not sure if all windows guests have the virtio rng/serial drivers installed.
This needs to be verified first. if not, there will be lots of failures in our tests.

File: provider/v2v_vmcheck_helper.py [hunk 2]
```diff
@@ -821,7 +824,7 @@ def check_windows_vm(self):
                     err_msg = "Driver %s not found" % driver
                     LOG.error(err_msg)
             expect_drivers = check_drivers[:]
-            if not expect_drivers:
+            if not expect_drivers and re.search(r"(Intel|AMD) Processor", win_dirvers).group(0) in cpu_drivers:
```
Comment (xiaodwan) [inline provider/v2v_vmcheck_helper.py:827]: I don't understand the 'and' expression here. If it's for the check of Processor, then put it to other places.
Here is not a good place for a check.

**Comment by xiaodwan** [comment]
You're changing the public logic for all windows guests. It may break our current tests. 
It's better to trigger our jobs to verify your patch.

File: provider/v2v_vmcheck_helper.py (hunk 1 above)
Comment (mxie91) [reply provider/v2v_vmcheck_helper.py:793]: Actually, virtio-win has added virtio balloon, virtio rng, virtio serial for all windows guests, so v2v needs to check if these drivers are installed normally after conversion,  but I found the installation of virtio balloon is not stable in v2v auto testing, so I didn't add it this time, will confirm if this a bug

# ls /usr/share/virtio-win/drivers/by-driver/Balloon/
2k12  2k12R2  2k16  2k19  2k22  2k8R2  w10  w11  w7  w8  w8.1

# ls /usr/share/virtio-win/drivers/by-driver/viorng/
2k12  2k12R2  2k16  2k19  2k22  2k8R2  w10  w11  w7  w8  w8.1

# ls /usr/share/virtio-win/drivers/by-driver/vioserial/
2k12  2k12R2  2k16  2k19  2k22  2k8R2  w10  w11  w7  w8  w8.1

File: provider/v2v_vmcheck_helper.py (hunk 2 above)
Comment (mxie91) [reply provider/v2v_vmcheck_helper.py]: Already remove this,  it's enough to use  'if re.search(r"(Intel|AMD) Processor", win_dirvers).group(0) not in cpu_drivers:'  to check, thanks

File: provider/v2v_vmcheck_helper.py (hunk 1 above)
Comment (xiaodwan) [reply provider/v2v_vmcheck_helper.py:793]: Can the drivers be installed successfully on all the guests? 
I can't remember clearly if we have such a bug which blocks the driver installation.

Comment (mxie91) [reply provider/v2v_vmcheck_helper.py]: I use jenkins job to test all windows guests,  all guests passed excepted win2016 guest failed with bug https://issues.redhat.com/browse/RHEL-17685(I just filed),  but I will re-pull the PR to improve the error info of cpu driver

https://libvirt-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/v2v/view/RHEL-9.4/job/v2v-RHEL-9.4-runtest-x86_64-signoff-libvirt/114/testReport/rhel/convert_vm_to_libvirt/

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**